src/main/java/org/springframework/data/couchbase/config/AbstractCouchbaseConfiguration.java

```diff
@@ -140,7 +145,22 @@ public ClusterEnvironment couchbaseClusterEnvironment() {
 	 * @param builder the builder that can be customized.
 	 */
 	protected void configureEnvironment(final ClusterEnvironment.Builder builder) {
+		Jackson2ObjectMapperBuilder jacksonBuilder = new Jackson2ObjectMapperBuilder();
```

Comment:
Actually I don't think this is how it should work? We do not expose jackson directly to the user, and all the encoding and decoding happens on the spring side and not with the SDK itself.. shouldn't this go into the mapper/converter logic instead?

Reply:
pinging @mp911de